.github/workflows/docker.yml

```diff
@@ -0,0 +1,24 @@
+name: Docker
+on:
+  # Scheduled workflows run on the latest commit on the default or base branch
+  schedule:
+    - cron: '20 2 * * *' # Daily at 02:20
```

Comment:
Why do we need this schedule? Wouldn't on push be sufficient for the tags matching below?

Reply:
Push is sufficient for the tags. The schedule is here to build a `latest` tag every day from the dev branch.

Reply:
We could leave it out for now though

Reply:
Ah of course, no that's fine for me